src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,69 @@
+package frc.robot.subsystems.shooter;
+
+public class ShooterConstants {
+
+  private static final String CONSTRUCTOR_EXCEPTION = "constant class";
+
+  private ShooterConstants() {
+    throw new IllegalStateException(CONSTRUCTOR_EXCEPTION);
+  }
+
+  public static final boolean DEBUGGING = false;
+  public static final boolean TESTING = false;
+  public static final String SUBSYSTEM_NAME = "Shooter";
+
+  public static final int RIGHT_SHOOTER_MOTOR_ID = 0;
```

Comment:
Capture CAN IDs and DIO channels in the 2024 Robot E. Tracker tab of the Software Feature sheet.